src/dynamic_programming/profile-dynamics.md

```diff
@@ -4,32 +4,36 @@
 
 Common problems solved using DP on broken profile include:
 
-* finding number of ways to fully fill an area (e.g. chessboard/grid) with some figures (e.g. dominoes)
-* finding a way to fill an area with minimum number of figures
-* finding a partial fill with minimum number of unfilled space (or cells, in case of grid)
-* finding a partial fill with the minimum number of figures, such that no more figures can be added
+- finding number of ways to fully fill an area (e.g. chessboard/grid) with some figures (e.g. dominoes)
+- finding a way to fill an area with minimum number of figures
+- finding a partial fill with minimum number of unfilled space (or cells, in case of grid)
+- finding a partial fill with the minimum number of figures, such that no more figures can be added
 
 ## Problem "Parquet"
 
 **Problem description.** Given a grid of size $N \times M$. Find number of ways to fill the grid with figures of size $2 \times 1$ (no cell should be left unfilled, and figures should not overlap each other).
 
-Let the DP state be: `D[i][mask]`, where `i = 1 ... N`, and `mask = 0 ... 2^M-1`. `i` respresents number of rows in the current grid, and `mask` is the state of last row of current grid. If `j`-th bit of `mask` is 0 then the corresponding cell is filled, otherwise it is unfilled. Clearly, the answer to the problem will be `D[N][0]`.
+Let the DP state be: $dp(i, mask)$, where $i = 1, \ldots N$ and $mask = 0, \ldots 2^{M-1}$.
```

Comment:
There is no function `dp(i, m)` around, while there is an array `dp[i][m]`. Please, do not "improve" formulae like that.

Reply:
I have changed it back to the DP array notation. Please take a look